frontend/src/pages/modelServing/screens/projects/DetailsPageMetricsWrapper.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { getInferenceServiceDisplayName } from '../global/utils';
+import MetricsPage from '../metrics/MetricsPage';
+import { ModelServingMetricsProvider } from '../metrics/ModelServingMetricsContext';
+import { getInferenceServiceMetricsQueries } from '../metrics/utils';
+import NotFound from '../../../NotFound';
+import { ProjectDetailsContext } from '../../../projects/ProjectDetailsContext';
+import { getProjectDisplayName } from '../../../projects/utils';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+
+const DetailsPageMetricsWrapper: React.FC = () => {
+  const { namespace: projectName, inferenceService: modelName } =
+    useParams<{ namespace: string; inferenceService: string }>();
+  const {
+    currentProject,
+    inferenceServices: { data: models, loaded },
+  } = React.useContext(ProjectDetailsContext);
+  const inferenceService = models.find(
+    (model) => model.metadata.name === modelName && model.metadata.namespace === projectName,
+  );
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+  if (!inferenceService) {
+    return <NotFound />;
+  }
+  const queries = getInferenceServiceMetricsQueries(inferenceService);
+
+  return (
+    <ModelServingMetricsProvider queries={queries}>
+      <MetricsPage
+        title={`${getInferenceServiceDisplayName(inferenceService)} metrics`}
+        breadcrumbItems={[
+          { label: 'Data Science Projects', link: '/projects' },
+          {
+            label: getProjectDisplayName(currentProject),
+            link: `/projects/${currentProject.metadata.name}`,
+          },
+        ]}
```

Comment:
Does the mocks really just stop and not show the current page? That seems... not right.

Reply:
@andrewballantyne What mocks are you talking about?

Reply:
I was lazy -- this was a question for @DaoDaoNoCode -- I looked at your Model Serving mocks. The breadcrumbs need one more value @DaoDaoNoCode -- the current page in a non link fashion.

I get you're doing it inside with the title -- but it's not looking accurate to @vconzola's mocks: https://www.sketch.com/s/113593f8-5970-49d6-a352-709b07639127/a/agOKG1D

Reply:
Ahh, right. Breadcrumbs should include the current page as plain (non-linkable) text.